Bottom: b8244af4f2a379a6dc0be26106cb5a19dbd12459
Top:    9315334268451cc52518aa4770e52cb3c898b8bd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-28 17:46:46 +0000

Refresh of tmp

---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5a7a260..628a4fa 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1371,7 +1371,7 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
             }
 
             printf ("Damnit\n");
-#if 1
+
             /* We need to find the contour before it, so we can update its next pointer */
             prev = container;
             while (prev->next != info.result) {
@@ -1384,7 +1384,6 @@ InsertHoles (jmp_buf * e, POLYAREA * dest, PLINE ** src)
             /* Add hole as the next on the list to be processed in this very function */
             info.result->next = *src;
             *src = info.result;
-#endif
           }
           /* End check for kicked out holes */
